h2/src/main/org/h2/store/PageStore.java

```diff
@@ -2028,4 +2028,8 @@ public synchronized void setBackup(boolean start) {
         backupLevel += start ? 1 : -1;
     }
 
+    public synchronized void setMaxMemory(int size) {
```

Comment:
can that method have more specific name like "setMaxCacheMemory"

Reply:
Good point, I will do that.